test/e2e/pages/allocation-cancel.js

```diff
@@ -0,0 +1,18 @@
+import { Selector } from 'testcafe'
+
+import Page from './page'
+
+class AllocationCancelPage extends Page {
+  constructor() {
+    super()
+    this.fields = {
```

Comment:
This (`this.fields`) feels like a copy/paste from another file but now doesn't really fit. These elements aren't _fields_. This object was used in other page objects to contain the fields on a form page.

Elsewhere we have use `this.nodes` to contain other elements on a page.